From ed77d478637c13c935def19282fc8975c2aa4c2a Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 18 Sep 2023 14:34:46 +0100
Subject: [PATCH 2/4] setup.py: don't generate version info in build

this should have already been generated by the time conda gets involved
---
 setup.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/setup.py b/setup.py
index 70422e03b..eea0df911 100755
--- a/setup.py
+++ b/setup.py
@@ -107,6 +107,14 @@ class clean(_clean):
 def get_version_info():
     """Get VCS info and write version info to version.py.
     """
+    # for conda builds, use the bundled version file
+    try:
+        from pycbc import version
+    except ModuleNotFoundError:
+        pass
+    else:
+        return version.version
+
     from pycbc import _version_helper
 
     class vdummy(object):
-- 
2.39.2

